src/subscript/config_jobs/PARAMS2CSV

```diff
@@ -0,0 +1,6 @@
+EXECUTABLE  params2csv
+
+DEFAULT  <OUTPUT> parameters.csv
+DEFAULT  <FILENAMECOLUMN> filename
+
+ARGLIST  "--filenamecolumnname" <FILENAMECOLUMN> "-o" <OUTPUT> <PARAMETERFILES> "--keepconstantcolumns"
```

Comment:
It occurred to me as well that maybe `--verbose` should go in here by default as well? What do you think?

Reply:
Totally support that.